hw02/PlayerController.cs

```diff
@@ -34,25 +48,43 @@ void Update()
         // TODO: Problem 4: What is an if statement? Write a couple of if statements to move the player with the WASD keys. 
         // We want to transform the player's position along the vertical and horizontal axis then multiply it by 
         // the corresponding variables we defined earlier. Lastly, we will need to multiply it all by Time.DeltaTime
+
+        if (Input.GetAxis("Vertical") != 0)
+        {
+            float translation = Input.GetAxis("Vertical") * speed * Time.deltaTime;
+            transform.Translate(0, 0, translation);
+        }
+
+        if (Input.GetAxis("Horizontal") != 0)
+        {
+            float translation = Input.GetAxis("Horizontal") * speed * Time.deltaTime;
+            transform.Translate(translation, 0, 0);
+        }
+
     }
 
     // TODO: Problem 5: What do the lines here do? Please provide inline comments and describe what each line is doing
     // What do Header, SerializeField, and Tooltip mean? 
-    [Header("General Setup Settings")]
-    [SerializeField] private InputAction movement;
+
+    // "Header" makes this text appear formatted as a header (large, bold) in the inspector
+    [Header("General Setup Settings")] 
```

Comment:
That's correct! Those are property attributes that helps anyone understanding the variables in the inspector.